public/css/style.css

```diff
@@ -0,0 +1,655 @@
+  font-size: 16px; }
+
+a {
+  text-decoration: none;
+  color: #000000; }
+
+img {
+  max-width: 100%; }
+
+.wrapper {
+  width: 80%;
+  margin: 0px auto; }
+
+.arrow {
+  border: solid #fff;
+  border-width: 0 2px 2px 0;
+  display: inline-block;
+  float: right;
+  padding: 6px; }
+
+header {
+  box-shadow: 0px 1px 11px #88888840; }
+  header .main_header {
+    display: flex;
+    padding: 0px;
+    min-height: 50px; }
+    header .main_header .main_header__logo {
+      flex: 1; }
+    header .main_header #m_menu {
+      display: none; }
+    header .main_header .main_header_logo {
+      flex-basis: 20%; }
+    header .main_header .nav {
+      flex-basis: 80%;
+      display: flex;
+      width: 100%;
+      position: relative; }
+    header .main_header ul li {
+      list-style: none;
+      padding: 5px 5px;
+      display: inline; }
+    header .main_header .menu {
+      width: 100%; }
+    header .main_header .menu_one {
+      width: 50%;
+      padding: 6% 0% 0% 5%;
+      display: inline-block; }
+    header .main_header .menu_two {
+      width: 50%;
+      text-align: right;
+      float: right;
+      font-size: 12px;
+      margin-top: 5px; }
+    header .main_header .cart-box {
+      position: absolute;
+      right: 0;
+      bottom: 0;
+      display: block; }
+    header .main_header .cart {
+      text-decoration: none;
+      padding: 15px 35px 15px 56px;
+      background-color: #eeeeee;
+      line-height: 1;
+      position: relative;
+      display: inline-block; }
+    header .main_header .cart-icon {
+      display: inline-block;
+      position: absolute;
+      left: 16px;
+      top: 9px; }
+      header .main_header .cart-icon span {
+        display: none; }
+    header .main_header .m-cart-items {
+      display: none; }
+
+.left-sidebar ul {
+  display: grid; }
+
+.footer {
+  display: flex;
+  background-color: #eeeeee; }
+  .footer .footer_copyright {
+    flex: 1;
+    padding: 1% 0%; }
+
+/*form style end*/
+/*form style start Login*/
+.main-login-container, .main-signup-container {
+  padding: 20px; }
+  .main-login-container .login-container, .main-login-container .signup-container, .main-signup-container .login-container, .main-signup-container .signup-container {
+    display: flex;
+    flex-wrap: wrap; }
+    .main-login-container .login-container article, .main-login-container .signup-container article, .main-signup-container .login-container article, .main-signup-container .signup-container article {
+      padding: 10px;
+      flex-basis: 100%; }
+    .main-login-container .login-container .login-box, .main-login-container .login-container .signup-box, .main-login-container .signup-container .login-box, .main-login-container .signup-container .signup-box, .main-signup-container .login-container .login-box, .main-signup-container .login-container .signup-box, .main-signup-container .signup-container .login-box, .main-signup-container .signup-container .signup-box {
```

Comment:
Same as above

Reply:
i manage the nesting selection upto 3 levels